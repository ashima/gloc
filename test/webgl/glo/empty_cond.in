#if defined E
#else
#endif

void main() {}
