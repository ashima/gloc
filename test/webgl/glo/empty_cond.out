Fatal error: exception Pp_lib.ParserError("fusing empty pptok_expr list")
Raised at file "glol.ml", line 51, characters 17-20
Called from file "gloc_lib.ml", line 284, characters 15-39
Re-raised at file "gloc_lib.ml", line 284, characters 56-57
Called from file "gloc.ml", line 140, characters 34-62
Called from file "list.ml", line 69, characters 12-15
Called from file "gloc.ml", line 145, characters 4-44
Called from file "gloc_posix.ml", line 9, characters 10-75
